tests/Unit/Training/WaitingList/WaitingListAccountTest.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Tests\Unit\Training\WaitingList;
 
+use App\Models\Atc\Endorsement\Condition;
```

Comment:
I don't see this import used in this file?

Reply:
https://github.com/VATSIM-UK/core/pull/2184/commits/a02fbed20ffe8c985c40cba1fce6b0937aacd75d